tsdb/engine/tsm1/wal.go

```diff
@@ -106,6 +105,13 @@ func NewWAL(path string) *WAL {
 			map[string]string{"path": path, "database": db, "retentionPolicy": rp},
 		),
 	}
+	w.SetLogOutput(os.Stderr)
+	return w
+}
+
+// SetLogOutput sets the location that logs are written to.
+func (l *WAL) SetLogOutput(w io.Writer) {
+	l.logger = log.New(w, "[tsm1wal] ", log.LstdFlags)
```

Comment:
`needs` is a bit strong; either it needs a `Lock` or a note that it should be called before the WAL is opened 😄 .